src/d3d9/d3d9_device.cpp

```diff
@@ -3865,7 +3869,11 @@ namespace dxvk {
     enabled.core.features.sampleRateShading = VK_TRUE;
     enabled.core.features.samplerAnisotropy = supported.core.features.samplerAnisotropy;
     enabled.core.features.shaderClipDistance = VK_TRUE;
+#ifdef DXVK_PLATFORM_DARWIN
+    enabled.core.features.shaderCullDistance = supported.core.features.shaderCullDistance;
+#else
```

Comment:
I dont think we even use this, so feel free to just do this anyway.

Reply:
Done